### DISCUSSION TITLE
Identical/similar icons

### DISCUSSION THREAD

**Discussion by yandere-sliver** [andrewgioia/keyrune#252] [General]
I do think the shooting star icons for Media Insert (pmei) and 2016 Heroes (htr) are absolutely identical.

Also MtG Promo (pmtg1), MtG Promo (Alt) (pmtg2) and Duels of the Planeswalkers (dpa) all use the same M in a circle motive. Are these really meant to be different icons could these be merged to make things a bit less cluttered in the Promo section?

**Comment by fenhl** [comment]
I know that at least the dpa symbol being different from the usual logo is intentional, as that reflects the symbol on the cards. I'd imagine the two other promo symbols also appear on different promo cards.

**Comment by andrewgioia** [comment]
It does appear that PMEI and HTR are identical, I'll take a look at those and any high-res scans to check again. If they're the same icon we can alias them, they don't need two separate glyphs.

I agree with @fenhl in that PMTG1, PMTG2, and DPA are all slight variations on the "M in a circle" that I believe were all used on separate prints. If someone has any other information on them I'd be happy to look more into those as well though.

**Comment by andrewgioia** [comment]
Moving this (and all older v4 issues) to discussions!
